docs/framework/windows-workflow-foundation/samples/using-the-invokemethod-activity.md

```diff
@@ -14,9 +14,9 @@ ms.author: "erikre"
 manager: "erikre"
 ---
 # Using the InvokeMethod Activity
-This sample demonstrates how to use the <!--zz <xref:System.Activities.Statements.InvokeMethod%601> -->[<code>System.Activities.Statements.InvokeMethod\`1</code>](https://msdn.microsoft.com/library/dd647677.aspx) activity to invoke public methods in public classes. The <!--zz <xref:System.Activities.Statements.InvokeMethod%601> -->[<code>System.Activities.Statements.InvokeMethod\`1</code>](https://msdn.microsoft.com/library/dd647677.aspx) activity allows a workflow to call methods against objects, pass in parameters, get the return value, specify types for generic methods, and specify whether the method is synchronous or asynchronous. 
+This sample demonstrates how to use the <xref:System.Activities.Statements.InvokeMethod%601> activity to invoke public methods in public classes. The <xref:System.Activities.Statements.InvokeMethod%601> activity allows a workflow to call methods against objects, pass in parameters, get the return value, specify types for generic methods, and specify whether the method is synchronous or asynchronous. 
```

Comment:
I think this class was removed from the product after being added for a preview release. We might have to delete this topic.

Reply:
OK. Want me to revert the changes?

Reply:
If you could, please!